zdm/figures.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
```

Comment:
somehow I never noticed this file, can you add a quick note on what this file is for? Not that I have done the same thing for mine...